.github/workflows/test.yml

```diff
@@ -14,7 +14,7 @@ env:
 
 jobs:
   build:
-    name: Verify & Build
+    name: Build
```

Comment:
This is undoing the change from the previous commit?

Reply:
Ah yes, because I noticed that the maintainers have branch protection set up for the `Build` stage and renaming this causes it to break.